ssm/command.go

```diff
@@ -24,3 +24,18 @@ func (s *SSM) GetCommandInvocation(ctx context.Context, instanceId, commandId st
 	log.Debugf("got output describing SSM Command: %+v", out)
 	return out, nil
 }
+
+func (s *SSM) SendCommand(ctx context.Context, input *ssm.SendCommandInput) (*ssm.Command, error) {
+	if input == nil || aws.StringValue(input.DocumentName) == "" {
```

Comment:
I think just checking the input is enough here
```
if input == nil {
```

Reply:
Done.